charts/etos/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: etos
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.6
```

Comment:
What is the difference between version and appVersion? In other words: Which one should I increment when making new releases?

Does appVersion show the helm chart as out of date when looking at "rancher" for instance?

Reply:
I though the comments were pretty clear but to sum it up: char version is always ticked when something changers, app version is only ticket when new version of ETOS or any of its services is tagged.

Reply:
Okay, so a new version would be this?
appVersion: 1.1.0
version: 0.1.7

Reply:
or?
appVersion: 1.1.0
version: 1.1.0

Reply:
It depends on what has changed. Chart version typically follows sem-ver as per docs.

Reply:
But we could link the chart version to the overall version-scheme/strategy if we feel that is more managable.

Reply:
We use the same version strategy as the rest of ETOS.
https://etos.readthedocs.io/en/stable/versioning.html

This means that appVersion and version will be the same all the time as we always update the "etos" repository event on PATCH updates.